src/components/Comments/comment.css

```diff
@@ -0,0 +1,103 @@
+@import url("https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800");
+/* ----- Coments ----- */
```

Comment:
Typo
No need to have duplicate comments (on line 12 and 23), just write `/* --- Comment --- */` on line 2